gui/src/controls/textBlock.ts

```diff
@@ -165,10 +166,20 @@ module BABYLON.GUI {
 
             rootY += this._currentMeasure.top;
 
+            var maxLineWidth: number = 0;
+
             for (var line of this._lines) {
                 this._drawText(line.text, line.width, rootY, context);
                 rootY += this._fontOffset.height;
+
+                if (line.width > maxLineWidth) maxLineWidth = line.width;
             }
+
+            this.width = maxLineWidth + 'px';

```

Comment:
This must be under a boolean as people may not want it
height computation must only take in account the fontOffset.height

Reply:
Wondering if we should also add padding size in the width/height computations